mesop/components/audio/audio.ng.html

```diff
@@ -1,2 +1,6 @@
 <!-- always show controls, otherwise you can't interact with the audio. -->
-<audio controls [src]="config().getSrc()"></audio>
+<audio
+  controls
+  [src]="config().getSrc()"
+  [attr.autoplay]="config().getAutoplay()? 'autoplay' : null"
```

Comment:
Although using `'autoplay'` seems to work, I'd set this to `'true'` as it's clearer and what's specified by the standard: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/autoplay